ui-library-click-to-call/README.md

```diff
@@ -7,6 +7,24 @@ Calling Widget is a tool that is meant for allowing instant interaction, whether
 
 If you want to just try it out follow the instructions below. If you are interested in information on how to build it into your application and how this sample is constructed please checkout our [tutorial here](https://learn.microsoft.com/en-us/azure/communication-services/tutorials/calling-widget/calling-widget-overview).
 
+## Pre-requisites to running the application
+You will need in order to run and edit the code in this project a code editor and terminal, [VSCode](https://code.visualstudio.com/Download) is recommended.
```

Comment:
nit: I would change this to:
"To run and edit the code in this project, you'll require a code editor and terminal. We recommend using VSCode for optimal functionality."